core/org.wso2.carbon.user.api/src/main/java/org/wso2/carbon/user/api/UserStoreManager.java

```diff
@@ -437,4 +437,14 @@ void deleteUserClaimValues(String userName, String[] claims, String profileName)
      */
     Properties getDefaultUserStoreProperties();
 
-}
\ No newline at end of file
+    /**
+     * Get if the user store is a local user store or not. By default,  this returns true.
+     *
+     * @return boolean - Is user store local or not.
```

Comment:
`Returns true if the userstore is a local userstore.` WDYT? Also need to start from an uppercase letter

Reply:
Fixed https://github.com/wso2/carbon-kernel/pull/3087